nextflow.config

```diff
@@ -119,13 +119,14 @@ params {
     amp_hmmsearch_savetargets                        = false
     amp_hmmsearch_savedomains                        = false
 
-    amp_ampcombi_db                                  = null
+    amp_ampcombi_db                                  = 'DRAMP'
```

Comment:
maybe thiis shoul dbe `amp_ampcombi_db_id` to match with e.g. mmseqs parameters,  and just `amp_ampcombi_db` for the path, for consistency with the the other parameters in the pipeline

Reply:
Ok by me!